sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/PushedDownOperators.scala

```diff
@@ -27,6 +28,7 @@ case class PushedDownOperators(
     aggregation: Option[Aggregation],
     sample: Option[TableSampleInfo],
     limit: Option[Int],
-    sortValues: Seq[SortOrder]) {
+    sortValues: Seq[SortOrder],
+    pushedPredicates: Seq[Predicate]) {
```

Comment:
Why don't the old v2 filter need this?

Reply:
Because just `AdvancedScanBuilderWithV2Filter` and `AdvancedScanBuilderWithV2Filter` implements `SupportsPushDownV2Filters` and no test the pushed filter info.